hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+:func:`~hypothesis.internal.compat.get_type_hints` now accepts any hints present
+in `thing.__signature__`, not just instances of `type`. This allows for the use
+of types from the `typing` module.
```

Comment:
```suggestion
This patch improves :func:`~hypothesis.strategies.builds` and
:func:`~hypothesis.strategies.from_type` support for explicitly defined ``__signature__``
attributes, from :ref:`version 5.8.3 <v5.8.3>`, to support generic types from the
:mod:`python:typing` module.

Thanks to Rónán Carrigan for identifying and fixing this problem!
```

Rephrasing to avoid discussing Hypothesis implementation details, reference the previous patch, and add cross-links.  If you'd prefer not to be identified as patch author that's also fine, but if you're OK with it we do like to thank contributors :slightly_smiling_face: 

FYI - in .rst, single-backticks are *italics* or "roles" like links, rather than `monospace` font - the most annoying incompatibility with markdown!

Reply:
Looks good to me, thanks! I'm not really familiar with rst or sphinx so was semi-guessing :smile: